sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.Legacy.cs

```diff
@@ -830,30 +832,177 @@ internal static SingleCategoryClassifyResultCollection ConvertToSingleCategoryCl
 
         #endregion
 
-        #region Long Running Operations
+        #region Healthcare
+
+        internal static HealthcareEntity ConvertToHealthcareEntity(Legacy.HealthcareEntity entity) =>
+          new HealthcareEntity(
+              entity.Text,
+              entity.Category,
+              entity.Subcategory,
+              entity.ConfidenceScore,
+              entity.Offset,
+              entity.Length,
+              ConvertToEntityDataSource(entity.Links),
+              ConvertToHealthcareEntityAssertion(entity.Assertion),
+              entity.Name);
+
+        internal static HealthcareEntityAssertion ConvertToHealthcareEntityAssertion(Legacy.HealthcareAssertion assertion)
+        {
+            if (assertion == null)
+                return null;
+
+            return new HealthcareEntityAssertion(
+                ConvertToEntityConditionality(assertion.Conditionality),
+                ConvertToEntityCertainty(assertion.Certainty),
+                ConvertToEntityAssociation(assertion.Association));
+        }
+
+        internal static EntityConditionality? ConvertToEntityConditionality(Legacy.Models.Conditionality? legacyConditionality) =>
+            legacyConditionality switch
+            {
+                null => (EntityConditionality?)null,
```

Comment:
```suggestion
                null => null,
```
There shouldn't be a need for a cast here.

Reply:
Didn't realize this is a normal enum. thank you!